extensions/amp-video/0.1/validator-amp-video.protoascii

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # <amp-video>
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "AMP-VIDEO"
+  spec_name: "amp-video extension .js script"
+  mandatory_parent: "HEAD"
+  unique_warning: true
```

Comment:
So this protoascii is getting reversed from what we typically do. Typically a protoascii for an extension has the script tag first then the specific tag that uses what that script tag provides. The script tag is commented by `# custom-element` and the specific tag is commented as `# <custom-element>`. This currently has that backwards.

Also besides my previous comment here saying `unique_warning: true` should be `unique: true` still stands except it should be for the `SCRIPT` portion, not the `AMP-VIDEO` portion.